engine/core/src/main/java/org/datacleaner/descriptors/CompositeDescriptorProvider.java

```diff
@@ -48,6 +57,9 @@ public CompositeDescriptorProvider(DescriptorProvider delegate1, DescriptorProvi
 
     public CompositeDescriptorProvider(List<DescriptorProvider> delegates) {
         this.delegates = delegates;
+        serverChecker = new ServerChecker();
+        serverCheckerThread = new Thread(serverChecker);
```

Comment:
Let's make the thread a daemon thread, and let's also give it a name so that it is easy to identify when investigating thread dumps.